x-pack/plugin/esql/src/main/java/org/elasticsearch/xpack/esql/parser/EsqlBaseLexer.java

```diff
@@ -25,160 +25,158 @@ public class EsqlBaseLexer extends LexerConfig {
-      "CLOSING_METRICS_MULTILINE_COMMENT", "CLOSING_METRICS_WS", "CLOSING_METRICS_QUOTED_IDENTIFIER", 
-      "CLOSING_METRICS_UNQUOTED_IDENTIFIER", "CLOSING_METRICS_BY", "CLOSING_METRICS_PIPE", 
-      "CHANGE_POINT_PIPE", "CHANGE_POINT_ON", "CHANGE_POINT_AS", "CHANGE_POINT_DOT", 
-      "CHANGE_POINT_COMMA", "CHANGE_POINT_QUOTED_IDENTIFIER", "CHANGE_POINT_UNQUOTED_IDENTIFIER", 
-      "CHANGE_POINT_LINE_COMMENT", "CHANGE_POINT_MULTILINE_COMMENT", "CHANGE_POINT_WS", 
-      "INSIST_PIPE", "INSIST_IDENTIFIER", "INSIST_WS", "INSIST_LINE_COMMENT", 
-      "INSIST_MULTILINE_COMMENT", "FORK_LP", "FORK_PIPE", "FORK_WS", "FORK_LINE_COMMENT", 
-      "FORK_MULTILINE_COMMENT"
+      "RENAME_LINE_COMMENT", "RENAME_MULTILINE_COMMENT", "RENAME_WS", "SHOW_PIPE", 
+      "INFO", "SHOW_LINE_COMMENT", "SHOW_MULTILINE_COMMENT", "SHOW_WS"
     };
   }
   public static final String[] ruleNames = makeRuleNames();
 
   private static String[] makeLiteralNames() {
     return new String[] {
-      null, "'dissect'", "'drop'", "'enrich'", "'eval'", "'explain'", "'from'", 
-      "'grok'", "'keep'", "'limit'", "'mv_expand'", "'rename'", "'row'", "'show'", 
-      "'sort'", "'stats'", "'where'", "'lookup'", null, null, null, null, null, 
-      null, null, null, null, null, null, null, null, "'|'", null, null, null, 
-      "'by'", "'and'", "'asc'", "'='", "'::'", "':'", "','", "'desc'", "'.'", 
-      "'false'", "'first'", "'in'", "'is'", "'last'", "'like'", "'not'", "'null'", 
-      "'nulls'", "'or'", "'?'", "'rlike'", "'true'", "'=='", "'=~'", "'!='", 
-      "'<'", "'<='", "'>'", "'>='", "'+'", "'-'", "'*'", "'/'", "'%'", "'{'", 
-      "'}'", null, null, "']'", null, "')'", null, null, null, null, null, 
-      null, null, null, "'metadata'", null, null, null, null, null, null, null, 
-      null, "'as'", null, null, null, "'on'", "'with'", null, null, null, null, 
-      null, null, null, null, null, null, "'info'", null, null, null, null, 
-      null, null, null, null, null, null, null, null, null, "'join'", "'USING'"
+      null, null, null, null, null, "'enrich'", "'explain'", "'dissect'", "'eval'", 
+      "'grok'", "'limit'", "'row'", "'sort'", "'stats'", "'where'", null, "'from'", 
+      null, "'lookup'", null, null, null, null, null, "'mv_expand'", "'drop'", 
+      "'keep'", null, "'rename'", "'show'", null, null, null, null, "'on'", 
+      "'with'", null, null, null, null, null, null, null, null, null, null, 
+      null, null, null, null, "'|'", null, null, null, "'by'", "'and'", "'asc'", 
+      "'='", "'::'", "':'", "','", "'desc'", "'.'", "'false'", "'first'", "'in'", 
+      "'is'", "'last'", "'like'", "'not'", "'null'", "'nulls'", "'or'", "'?'", 
+      "'rlike'", "'true'", "'=='", "'=~'", "'!='", "'<'", "'<='", "'>'", "'>='", 
+      "'+'", "'-'", "'*'", "'/'", "'%'", "'{'", "'}'", null, null, "']'", null, 
+      "')'", null, null, null, null, null, "'metadata'", null, null, null, 
```

Comment:
@nik9000 sorry for the late-stage comment, but it's a little curious to me that we lost the literal names for `LP` and `OPENING_BRACKET`. It changes the syntax errors, for example

`row var = 1 not in`

used to result in

`mismatched input '<EOF>' expecting '('`

but now says

`mismatched input '<EOF>' expecting LP`

Reply:
Sorry - just noticed it. Glad it wasn't this.